src/app/providers/search/search.ts

```diff
@@ -61,4 +61,9 @@ export class SearchProvider {
       { observe: 'response' },
     ).pipe(timeout(this.appConfig.getAppConfig().requestTimeout));
   }
+  getReEmail(applicationReference: string) {
```

Comment:
Can you add a blank line above this please, also what does this return is it a hashed string? If so the return type can be `getReEmail(applicationReference: string) : Observable<string> {`

Reply:
Can we also rename the method to 

`getRegeneratedEmails`